logger/logger.go

```diff
@@ -13,47 +13,42 @@
 //  limitations under the License.
 
 // Package logger offers simple logging on GCE.
+// Events are logged to the serial console as well as the event log.
 package logger
 
 import (
 	"fmt"
 	"log"
-	"os"
 	"path/filepath"
 	"runtime"
+	"strings"
 
 	"github.com/tarm/serial"
+	"golang.org/x/sys/windows/svc/eventlog"
 )
 
 var (
-	// Log is the logger's log.Logger.
+	// Log is the serial logger's log.Logger.
 	Log         *log.Logger
```

Comment:
should we rename the Log to something like serialLogger so it is not a public var and more readable